.github/workflows/build-push-images-pr.yaml

```diff
@@ -4,15 +4,113 @@ on:
     branches:
       - develop
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - 3.7
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+
+      - name: Test installing development dependencies
+        working-directory: deployment/docker
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r REQUIREMENTS-dev.txt
+
+      - name: Run Flake8 test
+        run: flake8 .
```

Comment:
Lint job always runs first.
Avoid building anything for linting, so when it fails, it fails quickly.